open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/view-generator-connectors/virtualization-solution-connector/src/main/java/org/odpi/openmetadata/openconnectors/governancedaemonconnectors/viewgenerator/virtulaizationsolutionconnector/derby/DerbyConnector.java

```diff
@@ -0,0 +1,81 @@
+package org.odpi.openmetadata.openconnectors.governancedaemonconnectors.viewgenerator.virtulaizationsolutionconnector.derby;
+
+import org.odpi.openmetadata.frameworks.connectors.ConnectorBase;
+import org.odpi.openmetadata.frameworks.connectors.properties.AdditionalProperties;
+import org.odpi.openmetadata.frameworks.connectors.properties.ConnectionProperties;
+import org.odpi.openmetadata.frameworks.connectors.properties.EndpointProperties;
+import org.odpi.openmetadata.openconnectors.governancedaemonconnectors.viewgenerator.virtulaizationsolutionconnector.VirtualizationSolutionConnectorBase;
+import org.odpi.openmetadata.openconnectors.governancedaemonconnectors.viewgenerator.virtulaizationsolutionconnector.model.LogicTable;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DerbyConnector extends ConnectorBase implements VirtualizationSolutionConnectorBase{
```

Comment:
VirtualizationSolutionConnectorBase is an interface not a class so should not have "Base" in its name.
Since this is the view generation collection, a name such as ViewGenerationInterface may be more understandable.

There is a missing java class that is the connector base class for the view generator connector.  For example:

```
public class ViewGeneratorConnectorBase extends ConnectorBase implements ViewGenerationInterface
{
}
```

Then the DerbyConnector extends ViewGenerationConnectorBase.